include/nleobs.h

```diff
@@ -16,6 +16,7 @@ typedef struct nle_observation {
     int action;
     int done;
     char in_normal_game;     /* Bool indicating if other obs are set. */
+    int how_done;            /* If game is really_done, how it ended. */
```

Comment:
This could be a `game_end_types` but I think it's better to keep this file standalone.